lib/cast_vote_msg.js

```diff
@@ -1,22 +1,21 @@
 const Assert = require('assert-plus')
 const Vote = require('./vote_entity')
 const Shapes = require('./shapes')
-const { fetchProp } = require('./utils')
-const { ValidationError, NotFoundError } = require('./errors')
+const { NotFoundError } = require('./errors')
 const VoteStats = require('./vote_stats')
 const PollRating = require('./poll_rating')
 const Reply = require('./reply')
 
-module.exports = function (opts = {}) {
+module.exports = function (opts) {
   async function handler(msg, reply) {
```

Comment:
name this function cast_vote - also helps with debugging
this name will appear in the seneca logs as cast_vote_<int>
'handler' is too generic